cookbook/cache/form_csrf_caching.rst

```diff
@@ -0,0 +1,50 @@
+.. index::
+    single: Cache; CSRF; Forms
+
+Caching Pages that Contain CSRF Protected Forms
+===============================================
+
+CSRF - or `Cross-site request forgery`_ - is a method by which a malicious
+user attempts to make your legitimate users unknowingly submit data that
+they don't intend to submit. Fortunately, CSRF attacks can be prevented by
+using a CSRF token inside your forms.
```

Comment:
I don't think we should repeat this here. Instead, we should put something similar to this: https://github.com/symfony/symfony-docs/pull/4141/files#diff-da27b730a6f0f3e3f23c45088b6113a0R1770 and tell them that CSRF is going to cause issues when caching, hence this article.